src/templates/finops-hub/modules/dataExplorer.bicep

```diff
@@ -0,0 +1,134 @@
+//==============================================================================
+// Parameters
+//==============================================================================
+
+@description('Required. Name of the hub. Used to ensure unique resource names.')
+param hubName string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Resource tags.')
+param tags object = {}
+
+@description('Required. The name of the Azure Key Vault instance.')
+param keyVaultName string
+
+//------------------------------------------------------------------------------
+// Variables
+//------------------------------------------------------------------------------
+
+// Generate globally unique cluster name: 4-22 chars; lowercase letters and numbers
+var safeHubName = replace(replace(toLower(hubName), '-', ''), '_', '')
+var clusterName = replace('${take(safeHubName, 22)}', '--', '-')
+
+//==============================================================================
+// Resources
+//==============================================================================
+
+resource cluster 'Microsoft.Kusto/clusters@2022-12-29' = {
+  name: clusterName
+  location: location
+  tags: tags
+  sku: {
+    capacity: 1
+    name: 'string'
+    tier: 'string'
+  }
+  identity: {
+    type: 'string'
+    userAssignedIdentities: {}
+  }
+  properties: {
```

Comment:
Need help with all these properties...

Reply:
Bassel - please provide your input.

Reply:
There are many props, which of them isn't clear, you can leave the default values